man/org.freedesktop.home1.xml

```diff
@@ -105,6 +106,8 @@ node /org/freedesktop/home1 {
 };
     </programlisting>
 
+    <!--method Rebalance is not documented!-->
```

Comment:
we really need to do better at documenting these, can you add at least a line with a reference?